sts-mock/jwks-helper-script/publish_keys_to_s3.sh

```diff
@@ -9,7 +9,7 @@ if [ $? -gt 0 ]; then
   exit 1
 fi
 
-if [ $# != 1 ]; then
+if [ $# != 2 ]; then
   echo "Incorrect number of parameters supplied. Supply the stack name." 1>&2
```

Comment:
As there's 2 arguments now required should we update this line to ask them to supply the stack name and the env name?

Reply:
Yes, I will do that, thank you for pointing that out.